src/components/color_picker/color_picker.ts

```diff
@@ -56,6 +56,9 @@ css/* scss */ `
       border-radius: 50px;
       border: ${ITEM_BORDER_WIDTH}px solid #666666;
       padding: 0px;
+      display: flex;
```

Comment:
I still see the broken style on the button even with this change, but I realized that the issue is already present in version 16.0 so we should address it in 16.0 directly. I think we can get rid of these lines, then?

Reply:
Okay sure.